rabbitmq-ballerina/client.bal

```diff
@@ -20,8 +20,7 @@ import ballerina/uuid;
 # The Ballerina interface to provide AMQP Channel related functionality.
 public client class Client {
```

Comment:
This is not isolated ?

Reply:
Fixed in 8ab4015f421f2e3f1215beab585b3b09b1e59126